NEWS.md

```diff
@@ -5,6 +5,7 @@
 * added `seq2()` for sequence generation
 * adapt `halveString()` for patterns with more than 1 character
 * add messages to `mergeAttr()` if combinations of merge variables from one data set do not occur in the other data set
+* add messages to `mergeAttr()` if the two data.frames intended to merge contain additional common variables (beyond thy by-variables)

```

Comment:
LGTM
Nit: Maybe `data.frames` and `by` should be displayed as code via back ticks?

Reply:
ok, I've added